packages/expo-sensors/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unpublished
 
+- Added support for the light sensor on Android. ([#18225](https://github.com/expo/expo/pull/18225) by [bearkillerPT](https://github.com/bearkillerPT))
```

Comment:
Now I noticed it,

Please move it under "New features" section 😉 

Spacing here is perfect - one blank line above and one below 🤩 keep it this way